src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -378,7 +379,27 @@ public String getQuestionAdditionalInfoHtml(int questionNumber,
                 Slots.ADDITIONAL_INFO_ID, additionalInfoId,
                 Slots.QUESTION_ADDITIONAL_INFO, additionalInfo.toString());
     }
-
+    
+    private void sortForRecipient(Map.Entry<String, List<Integer>> entry,
+            Map<String, String> identityMap,
+            Map<String, List<Integer>> sortedStorageMap,
+            FeedbackSessionResultsBundle bundle) {
+        String participantIdentifier = entry.getKey();
+        String name = bundle.getNameForEmail(participantIdentifier);
+        String nameEmail = name + participantIdentifier;
+        
+        identityMap.put(nameEmail, participantIdentifier);
+        sortedStorageMap.put(nameEmail, entry.getValue());
+    }
+    
+    private void sortForOption(Map.Entry<String, List<Integer>> entry,
+            List<String> optionList,
+            Map<String, List<Integer>> sortedStorageMap) {
+        String option = optionList.get(Integer.parseInt(entry.getKey()));
+        
+        sortedStorageMap.put(option, entry.getValue());
+    }
+    
```

Comment:
Put the utility methods below the where they are used, not above. We read in top-down fashion.